python/kotekan/shared_memory_buffer.py

```diff
@@ -85,10 +85,10 @@ def __init__(self, shared_memory_name, view_size, comet_manager=None):
 
         self.shared_mem_name = shared_memory_name
         self.semaphore = posix_ipc.Semaphore(shared_memory_name)
-        shared_mem = posix_ipc.SharedMemory(shared_memory_name)
+        self.shared_mem_file = posix_ipc.SharedMemory(shared_memory_name)
```

Comment:
This will throw an `ExistentialError` if `SharedMemory(shared_memory_name)` does not exist (which happens if the reader tries to instantiate, before the writer has created the SharedMemory region).

We should possibly check for `ExistentialError`s, and throw a `SharedMemoryError` with the appropriate message.